### PR TITLE
zap-plugin release authorization permission request

### PR DESCRIPTION
**Pull request by JordanGS**
Created my authorization request to get the ball rolling, the repository was created but there are some issues since i am no longer the administrator of it. If someone could help that would be great.

https://github.com/jenkinsci/zap-plugin

If you need anything more please let me know, thanks 👍

### CODE REVIEW

**Comment by daniel-beck** [comment]
Wrong wiki URL in the POM will result in distribution being suppressed. Plugins need valid wiki URLs.

File: permissions/plugin-zap.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+name: "zap"
+paths:
+- "org/jenkinsci/plugins/zap"
```
Comment (daniel-beck) [inline permissions/plugin-zap.yml]: Why did you strip the hyphen from this? Note that only the package name cannot handle the dash.

**Comment by daniel-beck** [comment]
Other than surprising missing dashes in the POM, looks good. You can of course keep the broken "reverse URL" groupId in the POM if you like it, but looks accidental rather than deliberate.

**Comment by JordanGS** [comment]
@daniel-beck clarification please.

My actual path is `zap/src/main/java/org/jenkinsci/plugins/zap`

because i cannot have a hyphen as part of the path name.

The yml should however say

```
paths:
- "org/jenkins-ci/plugins/zap"
```

is that correct?

**Comment by daniel-beck** [comment]
@JordanGS 

Yes.

> Note that only the package name cannot handle the dash.

Package names and groupId are unrelated, technically. They just start typically with the same prefix based on a similar convention, and packages don't allow hyphens.

**Comment by JordanGS** [comment]
@daniel-beck updated, good clarification.

**Pull request merged**